specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11/searchservice.json

```diff
@@ -0,0 +1,4170 @@
+          "default": 255,
+          "maximum": 300,
+          "description": "The maximum token length. Default is 255. Tokens longer than the maximum length are split. The maximum token length that can be used is 300 characters."
+        }
+      },
+      "description": "Breaks text following the Unicode Text Segmentation rules. This tokenizer is implemented using Apache Lucene.",
+      "externalDocs": {
+        "url": "http://lucene.apache.org/core/4_10_3/analyzers-common/org/apache/lucene/analysis/standard/StandardTokenizer.html"
+      }
+    },
+    "UaxUrlEmailTokenizer": {
+      "x-ms-discriminator-value": "#Microsoft.Azure.Search.UaxUrlEmailTokenizer",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Tokenizer"
+        }
+      ],
+      "properties": {
+        "maxTokenLength": {
+          "type": "integer",
+          "format": "int32",
+          "default": 255,
+          "maximum": 300,
+          "description": "The maximum token length. Default is 255. Tokens longer than the maximum length are split. The maximum token length that can be used is 300 characters."
+        }
+      },
+      "description": "Tokenizes urls and emails as one token. This tokenizer is implemented using Apache Lucene.",
+      "externalDocs": {
+        "url": "http://lucene.apache.org/core/4_10_3/analyzers-common/org/apache/lucene/analysis/standard/UAX29URLEmailTokenizer.html"
+      }
+    },
+    "TokenFilter": {
+      "discriminator": "@odata.type",
+      "properties": {
+        "@odata.type": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string",
+          "externalDocs":{
+            "url": "https://docs.microsoft.com/rest/api/searchservice/custom-analyzers-in-azure-search#index-attribute-reference"
+          },
+          "description": "The name of the token filter. It must only contain letters, digits, spaces, dashes or underscores, can only start and end with alphanumeric characters, and is limited to 128 characters."
+        }
+      },
+      "required": [
+        "@odata.type", "name"
+      ],
+      "description": "Abstract base class for token filters.",
+      "externalDocs": {
+        "url": "https://docs.microsoft.com/rest/api/searchservice/Custom-analyzers-in-Azure-Search"
+      }
+    },
+    "AsciiFoldingTokenFilter": {
+      "x-ms-discriminator-value": "#Microsoft.Azure.Search.AsciiFoldingTokenFilter",
+      "allOf": [
+        {
+          "$ref": "#/definitions/TokenFilter"
+        }
+      ],
+      "properties": {
+        "preserveOriginal": {
+          "type": "boolean",
+          "default": false,
+          "description": "A value indicating whether the original token will be kept. Default is false."
+        }
+      },
+      "description": "Converts alphabetic, numeric, and symbolic Unicode characters which are not in the first 127 ASCII characters (the \"Basic Latin\" Unicode block) into their ASCII equivalents, if such equivalents exist. T
```

Comment:
Please add this required parameter to the examples.

Reply:
I can't tell from the context -- Which parameter are you referring to?